sdk/Lusid.Sdk.Tests/Utilities/TestDataUtilities.cs

```diff
@@ -36,45 +62,48 @@ public TestDataUtilities(ITransactionPortfoliosApi transactionPortfoliosApi)
             IInstrumentsApi instrumentsApi,
             IQuotesApi quotesApi,
             IComplexMarketDataApi complexMarketDataApi = null,
-            IConfigurationRecipeApi recipeApi = null)
+            IConfigurationRecipeApi recipeApi = null,
+            IAggregationApi aggregationApi = null,
+            IPortfoliosApi portfoliosApi = null)
         {
             _transactionPortfoliosApi = transactionPortfoliosApi;
             _instrumentsApi = instrumentsApi;
             _quotesApi = quotesApi;
             _complexMarketDataApi = complexMarketDataApi;
             _recipeApi = recipeApi;
+            _aggregationApi = aggregationApi;
+            _portfoliosApi = portfoliosApi;
         }
 
-        public string CreateTransactionPortfolio(string scope)
+        public string CreateTransactionPortfolio(string scope, DateTimeOffset? effectiveAt = null)
         {
             var uuid = Guid.NewGuid().ToString();
-            
             //    Effective date of the portfolio, this is the date the portfolio was created and became live.  All dates/times
             //    must be supplied in UTC
-            var effectiveDate = new DateTimeOffset(2018, 1, 1, 0, 0, 0, TimeSpan.Zero);
-            
+            var effectiveDate = (effectiveAt == null) ? new DateTimeOffset(2018, 1, 1, 0, 0, 0, TimeSpan.Zero) : effectiveAt;
```

Comment:
can this be replaced with a null coalesce?

Reply:
Will update